prog/postgres/postgres_resource_nexus.rb

```diff
@@ -55,12 +55,21 @@ def before_run
   label def start
     nap 5 unless server.vm.strand.label == "wait"
     register_deadline(:wait, 10 * 60)
+    bud self.class, frame, :trigger_txid_current_on_parent if postgres_resource.parent
     hop_create_dns_record
   end
 
+  label def trigger_txid_current_on_parent
+    postgres_resource.parent.server.vm.sshable.cmd("sudo -u postgres psql -At -c 'SELECT txid_current()'")
+    pop "triggered txid_current"
+  end
+
   label def create_dns_record
     dns_zone&.insert_record(record_name: postgres_resource.hostname, type: "A", ttl: 10, data: server.vm.ephemeral_net4.to_s)
-    hop_initialize_certificates
+
+    reap
+    hop_initialize_certificates if leaf?
```

Comment:
Why here rather than later? Does txid_current need to finish before moving on? It seems like you could be doing certificate work concurrent, too

Reply:
makes sense, moved it there.